src/main/java/com/jalasoft/jfc/controller/PdfConverterController.java

```diff
@@ -44,15 +44,24 @@
     private PathJfc pathJfc;
 
     // Constant upload file.
-    private static final String UPLOADED_FOLDER = "files/input/";
+    private final String uploadedFolder;
```

Comment:
is it correct uploadedFolder? or it is uploadedFile

Reply:
ok i fix it now